lib/features/booking/view_model/cubit/add_inspection_booking_cubit.dart

```diff
@@ -30,4 +31,30 @@ class AddInspectionBookingCubit extends Cubit<AddInspectionBookingState> {
       // TODO: Implement when there is no user
     }
   }
+
+  StartTime? _startTime;
+  // DateTime? _inspectionDate;
+  // int? _duration;
+
+  set startTime(StartTime? startTime) {
+    _startTime = startTime;
+    validateBookingInspectionButton();
+  }
+
+  StartTime? get startTime => _startTime;
+
+  //TO DO: Remove in story 395 - handle duration selection
```

Comment:
if it's to be removed later, let's just remove it here now

Reply:
already removed the comments